app/controllers/wpcc/your_details_controller.rb

```diff
@@ -48,6 +51,12 @@ def session_params
 
     def amend_session
       your_details_form_params.each { |key, value| session[key] = value }
+      session[:manually_opt_in] = manually_opt_in?
+    end
+
+    def manually_opt_in?
```

Comment:
Users are not required to enrol, but if their salary is within the specified range and they want to enrol, they can only do so by manually enrolling. Manual enrolment is not part of the tool.